tests/installation/partitioning_raid.pm

```diff
@@ -176,13 +176,33 @@ sub run {
     assert_screen 'partitioning_raid-hard_disks-selected';
 
     if (get_var("OFW")) {    ## no RAID /boot partition for ppc
-        send_key 'alt-p';
-        if (!get_var('UEFI')) {    # partitioning type does not appear when GPT disk used, GPT is default for UEFI
-            assert_screen 'partitioning-type';
-            send_key 'alt-n';
+        if (is_storage_ng) {
+            record_soft_failure 'bsc#1063844';    # Cannot add partition from menu option "hard disks"
+            send_key 'down';
+            assert_screen 'partitioning_raid-disk_vda-selected';
+            send_key 'alt-d';
+        }
+        else {
+            send_key 'alt-p';
+        }
+        if (!get_var('UEFI')) {                   # partitioning type does not appear when GPT disk used, GPT is default for UEFI
+            if (is_storage_ng) {
+                record_soft_failure 'bsc#1055743';    # No partitioning type page ATM
+            }
+            else {
+                assert_screen 'partitioning-type';
+                send_key 'alt-n';
+            }
         }
         assert_screen 'partitioning-size';
-        wait_screen_change { send_key 'ctrl-a' };
+        send_key 'alt-c' if is_storage_ng;
+        if (is_storage_ng) {
+            my $size_hotkey = 'alt-s';
+        }
+        else {
+            my $size_hotkey = 'ctrl-a';
```

Comment:
Let's rewrite this one at least as my $size_hotkey = is_storage_ng ? alt-s : ctrl-a;
Now I even wonder why is it ctrl-a and not alt-a =)

Reply:
Because before it wasn't a hotkey. The textbox was already selected, then they selecte all text ctrl-a.
Now you have first to focus alt-s, and by doing that the text is already selected.